include/SMaterialLayer.h

```diff
@@ -43,14 +43,38 @@ namespace video
 			"texture_clamp_mirror_clamp_to_edge",
 			"texture_clamp_mirror_clamp_to_border", 0};
 
+
+	//! Texture minification filter.
+	/** Used when scaling textures down. */
+	enum E_TEXTURE_MIN_FILTER {
+		//! Nearest-neighbor interpolation.
+		ETMINF_NEAREST = 0,
+		//! Linear interpolation.
+		ETMINF_BILINEAR,
+		//! Linear interpolation across mipmaps.
+		/** Is equivalent to ETMINF_BILINEAR if mipmaps are disabled.
+		Only available as a minification filter since mipmaps are only used
+		when scaling down. */
+		ETMINF_TRILINEAR,
+	};

```

Comment:
OpenGL has more options. Basically, you can set intra-mip-level and inter-mip-level interpolation independently.

Reply:
All options provided by OpenGL are now exposed.

Reply:
OpenGL has 6 different modes: https://www.khronos.org/opengl/wiki/Sampler_Object#Filtering